curated_transformers/tests/repository/test_hf_hub.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+
+from curated_transformers.repository import HfHubRepository
+
+TEST_LINES = ["Line 1", "Line 2 🐍", "🤖" "\n\n", ""]
+
+
+@pytest.mark.slow
```

Comment:
I think we need another mark. We still want people to be able to run slow tests, even if they don't have access to our repo.

Reply:
Added a new marker for upload tests.